ui/src/router/details/RouterDetailsView.js

```diff
@@ -104,8 +101,8 @@ export const RouterDetailsView = ({ projectId, routerId, ...props }) => {
                 </RouterActions>
               </Fragment>
             )}
-            <EuiSpacer size="m" />
-
+          </EuiPageTemplate.Header>
+          <EuiPageTemplate.Section color={"transparent"}>
```

Comment:
Why do we need this `color={"transparent"}` ?

Reply:
Otherwise this happens 😅
![Screenshot 2022-08-30 at 7 12 00 PM](https://user-images.githubusercontent.com/36802364/187422264-212ac7ee-9020-47d6-a2d7-285489ac3145.png)